client/src/modules/vouchers/toolkit/advances_loans_installments/advances_loans_installments.modal.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>TREE.FINANCE</li>
+    <li class="title" translate>VOUCHERS.GLOBAL.ADVANCES_LOANS_INSTALLMENTS</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" novalidate>
+  <div class="modal-body">
+    <!-- cashbox -->
+    <bh-cashbox-select
+      cashbox-id = "ToolCtrl.searchQueries.cashbox_id"
+      name = "cashbox_id"
+      on-select-callback = "ToolCtrl.onSelectCashbox(cashbox)"
+      restrict-to-user = "false">
+    </bh-cashbox-select>
+
+    <bh-account-select
+      label="VOUCHERS.GLOBAL.DESTINATION_ACCOUNT"
+      on-select-callback="ToolCtrl.onSelectAccountCallback(account)"
+      exclude-title-accounts="true">
+      (<label translate>FORM.LABELS.DEBIT</label>)
```

Comment:
Can we filter by `account_type_id = 2`?   Should this always be a `liability`?

Reply:
I have a concern in Vanga the account he uses for social loans uses this account 
**510000 : Personal social loans**

Reply:
Hm... does that make sense?  Or is Vanga doing poor accounting?

Reply:
I see you changed it to both asset and liability - you are right, it can be both.  I was wrong.